configure.ac

```diff
@@ -48,6 +48,7 @@ AC_SUBST(X11_LIBS)
 PKG_CHECK_MODULES(GLIB, [glib-2.0 >= 2.28.0])
 AC_SUBST(GLIB_CFLAGS)
 AC_SUBST(GLIB_LIBS)
+AC_CHECK_LIB([glib-2.0], [g_get_os_info], [AC_DEFINE([HAS_G_GET_OS_INFO],[1],[glib supports g_get_os_info])])
```

Comment:
Instead of adding this define, you could use the `GLIB_CHECK_VERSION` macro as follows in your code:

```c
#if GLIB_CHECK_VERSION(2, 64, 0)
    // Do something
#endif
```

Reply:
Ah, I didn't realize that version checks are the preferred approach. I've changed that now.